sdks/java/core/src/main/java/org/apache/beam/sdk/io/PubsubUnboundedSource.java

```diff
@@ -1305,6 +1306,11 @@ public ProjectPath getProject() {
 
   @Nullable
   public TopicPath getTopic() {
+    return topic.get();
```

Comment:
Something seems off here. I know that `topic.get()` cannot return a `null` value. But `topic` is `@Nullable`, so maybe that is where the `null` should come from?